src/shared/hooks/useStaticContent.ts

```diff
@@ -0,0 +1,28 @@
+import { useEffect, useState } from 'react';
+
+import { useIntersectionObserver } from 'shared/hooks/useIntersectionObserver';
+
+export const useStaticContent = ({
+  targetSelector,
+  isStaticActive = false,
+}: {
+  targetSelector: string;
+  isStaticActive?: boolean;
+}) => {
+  const [isStatic, setStatic] = useState(isStaticActive);
```

Comment:
What do you think about using useState(false) here? Since isStaticActive can be undefined, we can set an initial state of false to handle this. Additionally, in the useEffect hook, we can add the following code: if (isStaticActive !== undefined) setStatic(isStaticActive);. This will update the static state only if isStaticActive is not undefined.

Reply:
Yes, exactly. Please check the latest commits, I set this variable to `false` by default